Formula/procyon-decompiler.rb

```diff
@@ -0,0 +1,32 @@
+class ProcyonDecompiler < Formula
+  desc "Modern decompiler for Java 5 and beyond"
+  homepage "https://github.com/mstrobel/procyon"
+  url "https://github.com/mstrobel/procyon/releases/download/v0.6.0/procyon-decompiler-0.6.0.jar"
+  sha256 "821da96012fc69244fa1ea298c90455ee4e021434bc796d3b9546ab24601b779"
+  license "Apache-2.0"
+
+  depends_on "openjdk"
+
+  def install
+    libexec.install "procyon-decompiler-#{version}.jar"
+    (bin/"procyon-decompiler").write <<~EOS
+      #!/bin/bash
+      export JAVA_HOME="${JAVA_HOME:-#{Formula["openjdk"].opt_prefix}}"
+      exec "${JAVA_HOME}/bin/java" -jar "#{libexec}/procyon-decompiler-#{version}.jar" "$@"
+    EOS
```

Comment:
Doesn't `write_jar_script` work?

Reply:
Yes, it does. It looks like a new Formula, but I used my old commit, which wrote its own script. Fixed it.